src/app/ApplicationTemplate.Shared.Views/Framework/Email/EmailService.Windows.cs

```diff
@@ -0,0 +1,38 @@
+﻿//-:cnd:noEmit
+#if __WINDOWS__
+using System;
+using System.Threading.Tasks;
+
+namespace ApplicationTemplate;
+
+public sealed partial class EmailService : IEmailService
+{
+	public Task Compose(Email email)
+	{
+		throw new NotImplementedException();
+
+//-:cnd:noEmit
+/* TODO: Use when Uno implementation is working and supporting attachments.
+ * See https://github.com/unoplatform/uno/issues/2432 for more details. */
+#if false
+		var emailMessage = new Windows.ApplicationModel.Email.EmailMessage()
+		{
+			Subject = email.Subject,
+			Body = email.Body,
+		};
+
+		foreach (var attachment in email.Attachments)
+		{
+			var file = await Windows.Storage.StorageFile.GetFileFromPathAsync(attachment.FullPath);
```

Comment:
Same wondering for `using` here

Reply:
Not here, it's in a false pragma if and it's not compiled, don't want any warning or visual underlining telling me that the namespace is not used.